doc/contribute/project_roles.rst

```diff
@@ -0,0 +1,279 @@
+* Responsibility to participate in community discussions.
+* Responsibility to mentor new contributors when appropriate
+* Responsibility to participate in the quality verification and release
+  process, when those happen.
+
+Maintainer
+##########
+
+A *Maintainer* is a Collaborator who is also responsible for knowing,
+directing and anticipating the needs of a given zephyr source code area.
+
+Maintainers have the following rights and responsibilities,
+in addition to those listed for Contributors and Collaborators:
+
+* Right to set the overall architecture of the relevant subsystems or areas
+  of involvement.
+* Right to make decisions in the relevant subsystems or areas of involvement,
+  in conjunction with the collaborators.
+* Responsibility to convey the direction of the relevant subsystem or areas to
+  the TSC
+* Responsibility to ensure all contributions of the project have been reviewed
+  within reasonable time.
+* Responsibility to enforce the code of conduct.
+
+
+Teams and Supporting Activities
+*******************************
+
+Assignee
+########
+
+An *Assignee* is one of the maintainers of a subsystem or code being changed.
+Assignees are set either automatically based on the code being changed or set
+by the other Maintainers, the Release Engineering team can set an assignee when
+the latter is not possible.
+
+* Right to dismiss stale reviews and seek reviews from additional maintainers,
+  developers and contributors
+* Right to block pull requests from being merged
+* Responsibility to re-assign a pull request if they are the original submitter
+  of the code
+* Responsibility to drive the pull request to a mergeable state
+* Solicit approvals from maintainers of the subsystems affected
+* Responsibility to drive the escalation process
+
+Release Engineering Team
+########################
+
+A team of active Maintainers involved in multiple areas.
+The team is appointed by the TSC and reports directly into the TSC.
+Release Engineering team has the following rights and responsibilities:
+
+* Right to merge code changes to the zephyr tree following the project rules.
+* Right to revert any changes that have broken the code base
+* Right to close any stale changes after <N> months of no activity
+* Responsibility to take directions from the TSC and follow them.
+* Responsibility to coordinate code merges with maintainers.
+* Responsibility to merge all contributions regardless of their
+  origin and area if they have been approved by the respective
+  maintainers and follow the merge criteria of a change.
+* Responsibility to keep the Zephyr code base in a working and passing state
+  (as per CI)
+
+
+Release Activity
+################
+
+.. figure:: img/img_release_activity.png
+     :width: 663px
+     :align: center
+     :alt: Release Activity
+
+
+MAINTAINERS File
+################
+
+Generic guidelines for deciding and filling in the Maintainers' list
+
+* The maintainers file shall coexist with the codeowner file.
```

Comment:
Should we use here the precise names of the files like you're doing in the heading of this section, i.e. `MAINTAINERS` and `CODEOWNERS`?

Reply:
Yes, that's a good point, actually, and does not change the content. I will apply the change.

Reply:
Fixed now